components/edit_post_modal/edit_post_modal.test.jsx

```diff
@@ -36,10 +36,10 @@ function createEditPost({canEditPost, canDeletePost, ctrlSend, config, license,
         title: 'test',
     };
     const actionsProp = actions || {
-        editPost: jest.fn(),
-        addMessageIntoHistory: jest.fn(),
-        hideEditPostModal: jest.fn(),
-        openModal: jest.fn(),
+        editPost: jest.fn((data) => new Promise((resolve) => resolve(data))),
```

Comment:
These can be simplified by using `Promise.resolve`
```suggestion
        editPost: jest.fn((data) => Promise.resolve(data)),
```
Or possibly even
```suggestion
        editPost: jest.fn(Promise.resolve),
```